.stickler.yml

```diff
@@ -0,0 +1,2 @@
+linters:
+  stylelint:
```

Comment:
`HINT`

Let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml) instead.

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).